extensions/notebook/src/telemetry.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import AdsTelemetryReporter from 'ads-extension-telemetry';
+
+const packageJson = require('../package.json');
+export const TelemetryReporter = new AdsTelemetryReporter(packageJson.name, packageJson.version, packageJson.aiKey);
+
+export const NotebookTelemetryView = 'Notebook';
+
+export enum NbTelemetryActions {
+	OpenNotebook = 'NotebookOpened',
+	OpenMarkdown = 'MarkdownOpened',
```

Comment:
There's a number of these such as OpenMarkdown that don't seem to be used. Either remove them or add the events.